frontend/packages/console-plugin-sdk/src/registry.ts

```diff
@@ -68,6 +79,10 @@ export class ExtensionRegistry {
     return this.extensions.filter(isFeatureFlag);
   }
 
+  public getCRFeatureFlags() {
```

Comment:
not necessarily tied to `CR` detection - it can be anything so we should come up with better name - something like `getCustomFeatureFlags`